openvm/src/powdr_extension/chip.rs

```diff
@@ -394,8 +394,7 @@ where
                 // Set the is_valid column to 1
                 row_slice[is_valid_index] = <Val<SC>>::ONE;
 
-                let evaluator =
```

Comment:
@georgwiese even if not in `evaI`, I suspect this might still make sense here, since this code is run for each row, for each bus interaction of the apc.